package.json

```diff
@@ -55,6 +55,7 @@
     "@reduxjs/toolkit": "^1.6.1",
     "big-integer": "^1.6.49",
     "bson": "4.4.1",
+    "deprecated-react-native-prop-types": "^5.0.0",
```

Comment:
Do we still need the `deprecated-react-native-prop-types` given that we've switched to `react-native-gesture-handler`?